routes/promotions.js

```diff
@@ -56,12 +67,14 @@ module.exports = db => {
     });
 
     // put to update a promotion request
-    router.put('/:promotionRequestId', async (req, res) =>{
+    router.put('/', async (req, res) =>{
```

Comment:
If a promotion request is approved, the user should get the credential as well. This is not implemented in the code rn. Needs fix